src/main/frontend/handler/shell.cljs

```diff
@@ -86,7 +86,8 @@
                                     [:hr]
                                     [:div.mb-2
                                      [:a.font-medium.mr-1.inline
-                                      {:on-click (fn [] (get-versioned-file-content hash path))}
+                                      {:on-click (fn [] ((notification/clear-all!)
+                                                         (get-versioned-file-content hash path)))}
```

Comment:
notification/show! has a fourth parameter to specify a unique modal id.
The id can be passed to notification/clear!.